src/components/footer/footer.style.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const Footer = styled.footer`
+  display: flex;
+  align-items: center;
+  font-size: 0.875rem;
+  height: 6.25rem; /* 100px */
+  margin: auto;
+  max-width: 61.25rem; /* 980px */
+  padding: 0 1.25rem; /* 20px */
+`;
+
+export const ProfyLink = styled.a`
```

Comment:
I see a small problem here because `flex: 1` is set to the links directly. A user might click into the empty space in the footer and be unexpectedly send to another page. You could wrap a div or so around the links to get around this

Reply:
Ah! Good catch. 😄